hypothesis-python/tests/conjecture/test_engine.py

```diff
@@ -778,6 +779,22 @@ def f(data):
     assert runner.exit_reason == ExitReason.max_iterations
 
 
+def test_exit_because_shrink_phase_timeout(monkeypatch):
+    def fast_time(val=[0]):  # noqa  # mutable-default-argument
```

Comment:
Using the default argument just to save putting this on the previous line seems kinda ugly, especially given the pile of comments it requires.